src/fs/patch.ts

```diff
@@ -66,22 +66,11 @@ function shimFs(binary: NexeBinary, fs: any = require('fs')) {
 
   const statTime = function () {
     return {
-      dev: 0,
-      ino: 0,
-      nlink: 0,
-      rdev: 0,
-      uid: 123,
-      gid: 500,
-      blksize: 4096,
       blocks: 0,
```

Comment:
Was this meant to be left?

Reply:
Yeah, I guess we could fake how many blocks the file takes up based on its size, but for now setting it to zero is fine.

Reply:
Maybe add a todo comment so this is documented? Prevents devs needing to find this conversation later on when they wonder why it was left like this.

Reply:
I added some best guess logic for that value.
I'm currently trying to find time to eliminate _all_ of this FS code.